run_chefspec.sh

```diff
@@ -7,4 +7,8 @@ $DIR/set_env.sh
 
 cd $WORKSPACE/repo
 rvm use 2.1.2
-bundle exec strainer test --only chefspec
+if [ -f Thorfile ]; then
```

Comment:
from the thor wiki:

"To create a thor task, the first thing you need is a file ending with the .thor extension (a file named Thorfile works as well)" 

we can change this if we want, I just went with Thorfile because I vaguely remembered that filename from when we first started using `kitchen init` and it would generate a 'Thorfile'